reference/physics.md

```diff
@@ -112,6 +112,9 @@ or stacking simulation, you will need to specify the [Physics](#physics) nodes
 for the boxes so that the friction and gravity forces are applied to these
 objects.
 
+Note that if you add a dynamic [Solid](#solid) node inside a passive object, then it will behave like a normal dynamic object independent from the parent one.
```

Comment:
The title of the section is "In passive objects".
I agree that this could not be the best place were to put this information because it is true not only for passive objects but also to active ones.
I can maybe move this sentence in a different section.